logger/logger.go

```diff
@@ -174,3 +175,12 @@ func debugEnabledOnKernelCmdline() bool {
 	m, _ := osutil.KernelCommandLineKeyValues("snapd.debug")
 	return m["snapd.debug"] == "1"
 }
+
+var timeNow = time.Now
+
+// StartupStageTimestamp produce snap startup timings message.
+func StartupStageTimestamp(stage string) {
```

Comment:
this is a bit different from what I proposed but is consistent with snap-confine. I don't care too much either way. So let it be like this and see if we ever have to do something similar but different somewhere else.